<launch>

    <node pkg="xbot_stereo_imu_sim" type="load_stereo_imu" name="load_stereo_imu" output="screen" />
    <param name="left_camera_topic" type="string" value="stereo_camera/left/image_raw"/>
    <param name="right_camera_topic" type="string" value="stereo_camera/right/image_raw"/>
    <param name="imu_topic" type="string" value="imu"/>

</launch>